uwblueprint/supportive-housing
# Log Records Seeding Script

## Implementation description
Created a bash script that can be ran to seed the log records table with an arbitrary amount of semi-random records

Add an env file in the seeding directory with these values:
```
#Log Records Table
LOG_RECORD_ROWS=50
EMPLOYEE_ID=1
ATTN_TO_ID=1
```


## Steps to test
1. Run `bash seed.sh` (if you're on Windows. include a -w flag)
2. Check your table to ensure entries are there


## What should reviewers focus on?
Whipped this up with little thought so there's probably room for improvement, but it at least serves as a solid starting point that can be built upon


## Checklist
